app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-event-tracker.js

```diff
@@ -51,8 +51,8 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         }
       }
 
-      // Ensure it only tracks aria-expanded in an accordion element, instead of in any child of the clicked element
-      if (target.closest('.gem-c-accordion')) {
+      // Ensure it only tracks aria-expanded in an accordion/super navigation header element, instead of in any child of the clicked element
+      if (target.closest('.gem-c-accordion') || target.closest('.gem-c-layout-super-navigation-header')) {
```

Comment:
I did start this, but I'm now concerned that this line is going to be slowly expanded to add further searches for specific classes, which feels brittle and unsustainable. I wonder instead if we could include an attribute on the element itself to trigger this behaviour?

Reply:
Good idea - maybe it checks for the presence of `data-ga4-expandable` ?

Reply:
Thanks @andysellick I have used a `data-ga4-expandable` attribute to make tracking `aria-expanded` more generic and not dependent on component classes. I didn't remove the `.gem-c-accordion` check as I didn't want to break the accordion tracking with this branch.

Reply:
Great stuff. I've added this to our tech notes to address separately for the accordion.